tests/integration/introduction/test_introduction.py

```diff
@@ -39,3 +40,58 @@ def test_intro_description_not_displayed(self):
 
         # Then description should not be displayed
         self.assertNotIn('qa-intro-description', content)
+
+    def test_ref_start_and_end_date(self):
```

Comment:
If we use a jinja filter to achieve the `$date to $date` logic we could unit test this logic instead of integration test